.devcontainer/Dockerfile

```diff
@@ -16,7 +16,9 @@
 FROM mcr.microsoft.com/devcontainers/base:1-ubuntu-22.04
 
 RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && export GO_VERSION=$(cat .go-version) \
```

Comment:
Doesn't seem to work

Reply:
It is indeed ineffective, and I have modified it to use the VERSION environment variable to specify the Go version for installation.